lib/get/get-full-source-space.js

```diff
@@ -94,5 +94,5 @@ function extractItems (response) {
 }
 
 function filterDrafts (items, includeDrafts) {
-  return includeDrafts ? items : items.filter((item) => !item.isPublished())
+  return includeDrafts ? items : items.filter((item) => !item.sys.publishedVersion)
```

Comment:
The filter logic is the other way around. It keeps items that passes the test. Since only published entries have `sys.publishedVersion` set, you have to remove the `!`